prow/cmd/deck/static/plugin-help-script.js

```diff
@@ -22,6 +22,7 @@ function redrawOptions() {
 
 window.onload = function () {
     // set dropdown based on options from query string
+    const hash = window.location.hash;
```

Comment:
If someone uses this link in a doc, or pastes it to someone, then we turn on more plugins and the page gets re-rendered, what happens to the viability of the link?

Reply:
as long as the id of the command does not change, it can be reachable